### DISCUSSION TITLE
Can not find account on chain by address

### DISCUSSION THREAD

**Discussion by xiegudong45** [NonceGeek/Move-Camp#191] [Q&A] [answered]
有谁遇到过这样的问题吗？启动starcoin节点后创建了一个账户地址，但是执行二进制文件时却报错了。
![image](https://user-images.githubusercontent.com/30063330/200101833-d085a4f7-9dd1-4a59-8d93-a920d7fcb43f.png)
![image](https://user-images.githubusercontent.com/30063330/200101858-adbb8c2c-3587-4c3a-8029-11d00cbe1599.png)

**Comment by Demian101** [comment, marked as answer]
看着过程没错，不应该啊。

检查下：
1.  $ account list   能不能显示账号
2. 检查下 $ mpm release 的输出地址

或者按照 https://github.com/NonceGeek/Web3-dApp-Camp/discussions/149 配置检查下

**Reply by peaceyourself** [reply]
新建的账号没有给测试代币，给了就ok了，参考 #200

**Reply by Demian101** [reply]
正解！
